template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/oozie/OozieRoleConfigProvider.java

```diff
@@ -30,19 +33,32 @@ public class OozieRoleConfigProvider extends AbstractRdsRoleConfigProvider {
 
     private static final String OOZIE_DATABASE_PASSWORD = "oozie_database_password";
 
+    private static final String OOZIE_DATABASE_JDBC_URL = "oozie.service.JPAService.jdbc.url";
+
     private static final String OOZIE_CONFIG_SAFETY_VALVE = "oozie_config_safety_valve";
 
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
         switch (roleType) {
             case OozieRoles.OOZIE_SERVER:
                 RdsView oozieRdsView = getRdsView(source);
-                List<ApiClusterTemplateConfig> config = new ArrayList<>(List.of(
-                    config(OOZIE_DATABASE_HOST, oozieRdsView.getHost()),
-                    config(OOZIE_DATABASE_NAME, oozieRdsView.getDatabaseName()),
-                    config(OOZIE_DATABASE_TYPE, oozieRdsView.getSubprotocol()),
-                    config(OOZIE_DATABASE_USER, oozieRdsView.getConnectionUserName()),
-                    config(OOZIE_DATABASE_PASSWORD, oozieRdsView.getConnectionPassword())));
+                String cmVersion = ConfigUtils.getCmVersion(source);
+                StringBuilder ozzieSafetyVale = new StringBuilder();
+                List<ApiClusterTemplateConfig> config = new ArrayList<>();
+                if (oozieRdsView.isUseSsl()
+                        && CMRepositoryVersionUtil.isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_2_2)) {
+                    ozzieSafetyVale.append(ConfigUtils
+                            .getSafetyValveProperty(OOZIE_DATABASE_JDBC_URL, oozieRdsView.getConnectionURL()));
+                } else {
+                    config.add(config(OOZIE_DATABASE_HOST, oozieRdsView.getHost()));
+                    config.add(config(OOZIE_DATABASE_NAME, oozieRdsView.getDatabaseName()));
+                }
+                config.add(config(OOZIE_DATABASE_TYPE, oozieRdsView.getSubprotocol()));
```

Comment:
Not a big deal but it would perhaps be sufficient to set `OOZIE_DATABASE_TYPE` in the `else` branch, i.e. when SSL is not enabled.